openpype/pipeline/colorspace.py

```diff
@@ -344,6 +345,12 @@ def get_imageio_config(
     imageio_global, imageio_host = _get_imageio_settings(
         project_settings, host_name)
 
+    # check if host settings are having enabled key and if it is False
+    if imageio_host.get("enabled") and imageio_host["enabled"] is False:
```

Comment:
This condition can never happen.
```suggestion
    if "enabled" in imageio_host and imageio_host["enabled"] is False:
```

or maybe explicit, to give a hint why it is like that
```
imageio_enabled = imageio_host.get("enabled")
if imageio_enabled  is None:
    imageio_enabled = True

if not imageio_enabled :
    return False
```

Reply:
hey @fabiaserra it should be as it is said `check if host settings are having enabled key and if it is False`

We want ot make sure this is only happening if the bool is explicity set to False and exists in settings group. To explain it more: some productions might already set project overrides with older settings and since imageio category is group enabled, new schema is not effecting it and the `enable` key will not be part of settings data.

Reply:
sure, but I don't see how that is any different from what I'm saying?
`imageio_host.get("enabled") == False:` will only evaluate to True if `"enabled"` key exists and it's equal to False.